shared/src/business/entities/cases/Case.ts

```diff
@@ -1785,20 +1788,14 @@ export class Case extends JoiValidationEntity {
     return this;
   }
 
-  generateTrialSortTags(): {
-    hybrid: string;
-    nonHybrid: string;
-  } {
-    return generateTrialSortTags(this);
-  }
-
   /**
    * set as calendared
    * @param {object} trialSessionEntity - the trial session that is associated with the case
    * @returns {Case} the updated case entity
    */
   setAsCalendared(trialSessionEntity) {
     this.updateTrialSessionInformation(trialSessionEntity);
```

Comment:
I could be totally missing something here, but aren't we setting `trialSessionId`, `trialDate`, etc. right above this line setting `addedToTrialSession = true`? And if so, couldn't we just filter by `trialSessionId` in our eligible cases query?

Reply:
Good heavens. You are not overlooking anything, I do not know how this was not apparent. Fantastic!